code/pizzabot.js

```diff
@@ -1,7 +1,62 @@
-const vegetarian = "Vegetarian Pizza"
-const hawaiian = "Hawaiian Pizza"
-const pepperoni = "Pepperoni Pizza"
-
-const pizzaPrice = 80
+const vegetarian = "Vegetarian Pizza";
+const hawaiian = "Hawaiian Pizza";
+const pepperoni = "Pepperoni Pizza";
+const pizzaPrice = 80;
 
 //Put your Javscript code here:
+
+const orderPizza = () => {
+  alert(
+    `Hey! Happy to serve your pizza. On our menu we have ${vegetarian}, ${hawaiian} and ${pepperoni}.`
+  );
+  const orderName = prompt(
+    "Enter the name of the pizza you want to order today."
+  );
+
+  const validateOrderName = () => {
+    if (
+      orderName === vegetarian ||
+      orderName === hawaiian ||
+      orderName === pepperoni
```

Comment:
Well done :) There are two things I would like to mention :

1. Have you considered what will happen if user types proper pizza name, but lowercase?
So rather `vegetarian pizza` than `Vegetarian Pizza`?
Right now, your code will not accept that answer. We can easly fix that, there is a nice trick for that : you can use method `.toLowerCase()` on both `orderName` and `vegetarian`, `hawaiian` and `pepperoni` - thanks to that we will be comparing values with only lower case letters. It would look something like that :

`if (orderName.toLowerCase() === vegetarian.toLowerCase() || ... )`.

To create even more clean code, we store these values inside local variables so :

`const orderNameLower = orderName.toLowerCase();`
`const vegetarianLower = vegetarian.toLoweCase();`
`...`

So in our if statement we can write :

`if (orderNameLower === vegetarianLower || ... )`.

This is clean code practice, please fix that.

2. In your `validateOrderName` function you are using global variable `orderName`. We should pass it into our function as an argument rather than reaching for it from function. This is another clean code practice. You just have to remember that if you declare you `validateOrderName` function with argument, you need to pass it in line 33, where you call your function. Fix that as well